e2eTests/scenarios/CustomTemplate/runtest.ps1

```diff
@@ -140,7 +140,7 @@ $customJobs = @(
     }
 )
 # Add custom Jobs
-$cicdYaml.AddCustomJobsToYaml($customJobs)
+$cicdYaml.AddCustomJobsToYaml($customJobs, [CustomizationOrigin]::TemplateRepository)
```

Comment:
Won't this add this customjobs + the "DO NOT EDIT" comment to the template repository? The comment probably shouldn't be in the template repo

Reply:
Changed the parameter to _FinalRepository_.